riscv-sbi.adoc

```diff
@@ -6,6 +6,9 @@
 include::autogenerated/revision.adoc-snippet[]
 :revremark: This document is stable. Assume anything could still change, but limited change should be expected. Visit http://riscv.org/spec-state for further details.
 :url-riscv: http://riscv.org
+:revdate: 8/2023
+:revnumber: v2.0-rc2-6-g35e723d9cfc
```

Comment:
revdate & revnumber are autogenerated from the makefile

Reply:
The latest changes seems to have just commented them out instead of removing them. Do you need it for github actions ?

Reply:
This is the outdated code.